autoread.js

```diff
@@ -45,6 +45,10 @@ function play(){
 function initSpeak(){
 	speech = new SpeechSynthesisUtterance();
 	addSpeakBox();
+
+	// Disable the "Enable Speech" button
+	let enableSpeechBtn = document.getElementById("enable_speech_btn");
+	enableSpeechBtn.disabled = "disabled";
```

Comment:
diabled is a boolean so set disabled = true

Reply:
Done.